.github/workflows/test.yaml

```diff
@@ -42,11 +42,15 @@ jobs:
         shell: bash -l {0}
         run: pip install --no-deps -e .
 
-      - name: Run tests
+      - name: Install pytest-xdist for parallel testing
```

Comment:
This is probably a bad idea, as oversupscription of CPUs and RAM will slow down tests.

Reply:
Ah I see. Okay, I'll revert the script to remove parallelization. Thanks for the heads-up.